testdata/book/httpbin_include.yml

```
@@ -0,0 +1,19 @@
+desc: testing include
+runners:
+  req:
+    endpoint: ${HTTPBIN_END_POINT:-https://httpbin.org/}
+vars:
+  jsonRequestBody:
+    foo: "test"
+    bar: 1
+  counter: 0
+steps:
+  -
+    req:
+      /post?count={{ vars.counter }}:
```

[NOTE]
I thought I could reference the loop counter directly in the include destination, but I couldn't.
I don't think it's a problem because there are alternatives as per the test scenario.
```suggestion
      /post?count={{ i }}:
```

---

> I thought I could reference the loop counter directly in the include destination, but I couldn't.

Yes. Because the location of the `i` variable is root, I thought it would conflict with the loop counter in the included.

I also thought `parent.i` would be uncool 😄